package.json

```diff
@@ -6,8 +6,8 @@
     "pnpm": "9"
   },
   "scripts": {
-    "prettier:check": "prettier --check **/*.{json,md} --ignore-path=.prettierignore",
-    "prettier:write": "prettier --write **/*.{json,md} --ignore-path=.prettierignore",
+    "prettier:check": "prettier --check \"**/*.{json,md,js,ts,yml,yaml}\" --ignore-path .prettierignore",
```

Comment:
why was the `=` removed here?

Reply:
Unintentional. Added back in b85afc670cbe22a42c4db9977441f37525514ca9.